src/test/java/uk/gov/pay/api/json/RequestJsonParserTest.java

```diff
@@ -162,6 +162,24 @@ public void parsePaymentRequest_whenDelayedCaptureFieldIsNotABoolean() throws Ex
         parsePaymentRequest(jsonNode);
     }
     
+    @Test
+    public void parsePaymentRequest_whenMotoFieldIsNotABoolean() throws Exception {
```

Comment:
Why has this test been re-added? It's already tested in CreateCardPaymentRequestDeserializerTest

Reply:
Apologies I thought I removed it. Done